actions/consul.go

```diff
@@ -28,7 +28,7 @@ func (cs *consulStart) Execute(m *model.Model) error {
 	return m.ForEachHost(cs.hostSpec, 24, func(c *model.Host) error {
 		ssh := lib.NewSshConfigFactory(c)
 
```

Comment:
General pattern is to put the output to a file somewhere. I can see using screen for debugging, not sure it makes sense for long term use. Thoughts?

Reply:
Screen was an internal recommendation to more consistently background these processes. Still unclear exactly why they don't get backgrounded 100% of the time by using &. I do agree on logging though. Completely slipped my mind for the mvp rush.